dangerzone/document.py

```diff
@@ -103,6 +103,31 @@ def default_output_filename(self) -> str:
 
     def announce_id(self) -> None:
         log.info(f"Assigning ID '{self.id}' to doc '{self.input_filename}'")
+    def set_output_dir(self, path: str) -> None:
+        if not self._output_filename:
+            self.set_default_output_filename()
```

Comment:
Most probably we shouldn't have any need for this function call, once we merge #216.

Reply:
I think we still will need. On the UI there is a button to change the output directory of all the files. And that should call this one.

Reply:
Ah, I see. In that case, we need a common function for this, since we also have the `default_output_filename` property that does a slightly different thing.